source/13_envs/acrobot_zh.rst

```diff
@@ -0,0 +1,160 @@
+Acrobot
+~~~~~~~~~~~~~~~~~~
+
+概述
+=======
+Acrobot机器人系统是强化学习中的经典控制问题。该系统包括两个关节和两个连杆，其中两个连杆之间的关节可以被驱动。系统的初始状态是向下悬挂的。目标是在驱动关节上施加力矩，使连杆的自由端摆动到指定高度。如下图所示。
+
+.. image:: ./images/acrobot.gif
+   :align: center
+   :scale: 80%
+安装
+====
+
+安装方法
+--------
+
+Acrobot 环境内置在 gym 中，直接安装 gym 即可。其环境 id 是\ ``Acrobot-v1`` \。
+
+.. code:: shell
+
+    pip install gym
+    
+验证安装
+--------
+
+在 Python 命令行中运行如下命令验证安装成功。
+
+.. code:: shell 
+
+    import gym
+    env = gym.make('Acrobot-v1')
+    obs = env.reset()
+    print(obs)  
+
+环境介绍
+=========
+
+动作空间
+----------
+
+Acrobot 的动作空间属于离散动作空间，有 3 个离散动作，分别是施加+1力矩, -1力矩和不施加力矩。
+
+-  \ ``-1的力矩`` \: 0 表示向关节施加-1的力矩。
+
+-  \ ``0的力矩`` \: 1 表示向关节施加0的力矩。
+
+-  \ ``+1的力矩`` \: 2 表示向关节施加+1的力矩。
+
+使用 gym 环境空间定义则可表示为：
+
+.. code:: python
+    
+    action_space = spaces.Discrete(3)
+
+状态空间
+----------
+
+Acrobot 的状态空间有 6 个元素，分别是：
+
+
+- \ ``Cosine of theta1`` \：第一个连杆和竖直方向角度的cos值，范围是 \ ``[-1, 1]`` \。
+  
+- \ ``Sine of theta1`` \：第一个连杆和竖直方向角度的sin值，范围是 \ ``[-1, 1]`` \。
+
+- \ ``Cosine of theta2`` \：第二个连杆相对于第一个连杆的角度的cos值，范围是 \ ``[-1, 1]`` \。
+
+- \ ``Sine of theta2`` \：第二个连杆相对于第一个连杆的角度的sin值，范围是 \ ``[-1, 1]`` \。
+
+- \ ``Angular velocity of theta1`` \：第一个连杆相对于竖直方向的角速度，范围是 \ ``[-4 * pi, 4 * pi]`` \。
+
+- \ ``Angular velocity of theta2`` \：第二个连杆相对于第一个连杆的角速度，范围是 \ ``[-9 * pi, 9 * pi]`` \。
+
+
+theta1 是第一个关节的角度，其中角度 0 表示第一个链接直接指向下方。
+
+theta2 是相对于第一个连杆的角度。 角度 0 对应于两个链接之间具有相同的角度。
+
+
+奖励空间
+-----------
+目标是让自由端以尽可能少的步数达到指定的目标高度，因此所有未达到目标的步数都会产生 -1 的奖励。 达到目标高度会导致终止，奖励为 0。
+
+
+终止条件
+------------
+Cartpole 环境每个 episode 的终止条件是遇到以下任何一种情况：
```

Comment:
Cartpole?

Reply:
typo mistake, check the whole article and changed it.